app/uk/gov/hmrc/personaldetailsvalidation/views/template/personal_details_postcode.scala.html

```diff
@@ -0,0 +1,95 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.personaldetailsvalidation.model.{PersonalDetails, CompletionUrl}
+@import uk.gov.hmrc.play.views.html.helpers._
+@import uk.gov.hmrc.personaldetailsvalidation.endpoints.routes
+@import uk.gov.hmrc.views.ViewConfig
+@import uk.gov.hmrc.views.html.components.date
+@import uk.gov.hmrc.views.html.template.{main_template, error_prefix}
+
+@import uk.gov.hmrc.personaldetailsvalidation.model.PersonalDetailsWithPostcode
+@(pageForm: Form[PersonalDetailsWithPostcode],
+        completionUrl: CompletionUrl)(
+        implicit request: Request[_],
+        messages: Messages,
+        viewConfig: ViewConfig
+)
+
+@main_template(title = error_prefix(pageForm) + messages("personal-details.title")) {
+
+    @if(pageForm.hasErrors) {
+        @errorSummary(
+            heading = Messages("error-summary.heading"),
+            form = pageForm
+        )
+    }
+
+
+    <h1 class="heading-xlarge">
+        <span class="heading-secondary">
+            @messages("personal-details.faded-heading")
+        </span>
+        @messages("personal-details.header")
+    </h1>
+
+
+    <p>@messages("personal-details.paragraph")</p>
+
+    @form(routes.PersonalDetailsCollectionController.submit(completionUrl, alternativeVersion = true), 'class -> "form js-form", 'autocomplete -> "off", 'novalidate -> "novalidate") {
+
+        <div class="form-group">
+        @input(
+            field = pageForm("firstName"),
+            '_label -> messages("personal-details.firstname"),
+            '_inputClass -> "input--medium input--cleared",
+            '_labelTextClass -> "form-label-bold",
+            '_labelClass -> "soft--ends"
+        )
+        </div>
+        <div class="form-group">
+        @input(
+            field = pageForm("lastName"),
+            '_label -> messages("personal-details.lastname"),
+            '_inputClass -> "input--medium input--cleared",
+            '_labelTextClass -> "form-label-bold",
+            '_labelClass -> "soft--ends"
+        )
+        </div>
+        <div class="form-group">
+            @input(
+                field = pageForm("postcode"),
+                '_label -> messages("personal-details.postcode"),
+                '_inputClass -> "input--small input--cleared",
+                '_labelTextClass -> "form-label-bold",
+                '_inputHint -> Html(Messages("personal-details.postcode.hint")),
+                '_labelClass -> "soft--ends"
```

Comment:
Shouldn't there be a link to go from postcode form to nino form? If discussed already, then ignore otherwise check with Victoria.